chcemvediet/apps/inforequests/tests/test_models/test_action.py

```diff
@@ -709,15 +774,20 @@ def test_by_email_and_by_smail_query_methods(self):
         self.assertItemsEqual(result_by_email, [request, confirmation, refusal, extension])
         self.assertItemsEqual(result_by_smail, [appeal, remandment, expiration])
 
-    def test_order_by_pk_query_method(self):
+    def test_of_inforequest_query_method(self):
         inforequest = self._create_inforequest()
         branch = self._create_branch(inforequest=inforequest)
-        actions = [self._create_action(branch=branch) for i in range(20)]
+        actions = [self._create_action() for _ in range(10)]
+        actions = [self._create_action(branch=branch) for _ in range(10)]
```

Comment:
Asi tie dve premenne nazvi rozne. Lebo je divne, ze najprv nieco priradis do `actions` a potom opat. Treba ich nejak rozlisit.

Reply:
@martinmacko47 pouzil som `_` pre nevyuzitu premennu. Alebo radsej zdoraznit vyznam tej premennej pridanim nazvu. Napr. `other_actions` a `of_inforequest_actions`.

Reply:
@martinmacko47 A teraz ked sa na to pozeram spatne, sa zamyslam ci to je vobec potrebne vytvarat actions, ktore neptaria pod danu inforequests. Najprv som chceml poukazt ze existuju aj ine actions v databaze, ze tie zle odfiltruje. Teraz vsak to povazujem za nadbytocne, alebo?

Reply:
@viliambalaz Je fajn otestovat, ze to neselectne akcie inej ziadosti. Asi to sprav tak, ze v teste spravis dve ziadosti `inforequest1` a `inforequest2`, kazdu so svojimi akciami. A potom overis, ze `of_inforequest` selectne akcie len jednej z nich.

Reply:
:ok: